ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -155,7 +155,8 @@ class LocalAssemblerData : public GroundwaterFlowLocalAssemblerInterface
         // fetch hydraulic conductivity
         ParameterLib::SpatialPosition pos;
         pos.setElementID(_element.getID());
-        auto const k = _process_data.hydraulic_conductivity(t, pos)[0];
+        auto const k = hydraulicConductivity<GlobalDim>(
```

Comment:
You can use MPL and formEigenTensor. With that the function of hydraulicConductivity can be removed.

Reply:
PR #2820 will change it.

Reply:
I see this might be a quick fixing.